indico/modules/designer/client/js/index.js

```diff
@@ -475,14 +475,10 @@ import {$T} from 'indico/utils/i18n';
     return !(top1 >= bottom2 || top2 >= bottom1);
   }
 
-  function normalizeZindex() {
-    var increament = 1;
-    Object.values(items)
-      .sort((item1, item2) => item1.zIndex - item2.zIndex)
-      .forEach(item => {
-        item.zIndex = increament + 9;
-        increament++;
-      });
+  function normalizeZIndex() {
+    _.sortBy(Object.values(items), 'zIndex').forEach((item, n) => {
+      item.zIndex = n + 10;
```

Comment:
Probably related to the bug I've pointed out: `normalizeZIndex` isn't actually setting any CSS properties.

Reply:
Yes, I think it may be because of this because moving the `normalizeZIndex` call does not fix the bug